### ISSUE TITLE
[BUG] [Linter] T002 errors when single quotes wrap attribute value with double quotes

### ISSUE THREAD

**Issue by ylac** [Riverside-Healthcare/djLint#392]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: e.g. macOS 12.5
 - Python Version: 3.10.6
 - djLint Version: 1.15.0
 - template language: e.g. Jinja

## Issue
The linter incorrectly flags HTML attribute values that have outer single quotes when the value contains double quotes.

## How To Reproduce
For example, `<div class="card" data-list='{"name": "blah"}'>` gets flagged.

The output looks like:

```
T002 1:0 Double quotes should be used in tags. {% extends "layout.h
```

Note, the line number is incorrect too.

## Contents of .djlintrc[tool.djlint]
```
{
  "ignore": "T003,H006,H013,H017,H021,T028,H030,H031",
  "profile": "jinja"
}
```

**Comment by christopherpickering** [comment]
Can you share a more complete html snippet?

**Comment by ylac** [comment]
Sure see my edit above 👍

**Comment by christopherpickering** [comment]
Do you have anything else in the file? I can't reproduce with just that line.
